pkg/odo/cli/version/version.go

```diff
@@ -23,58 +23,66 @@ var (
 	GITCOMMIT = "HEAD"
 )
 
+// RecommendedCommandName is the recommended version command name
+const RecommendedCommandName = "version"
+
+var versionLongDesc = ktemplates.LongDesc("Print the client version information")
+
 var clientFlag bool
 
-// versionCmd represents the version command
-var versionCmd = &cobra.Command{
-	Use:   "version",
-	Short: "Print the client version information",
-	Long:  "Print the client version information",
-	Example: `  # Print the client version of Odo
-  odo version
-	`,
-	Run: func(cmd *cobra.Command, args []string) {
+// NewCmdVersion implements the version odo command
+func NewCmdVersion(name, fullName string) *cobra.Command {
+	// Add a defined annotation in order to appear in the help menu
+
+	// versionCmd represents the version command
+	var versionCmd = &cobra.Command{
+		Use:   name,
+		Short: versionLongDesc,
+		Long:  versionLongDesc,
+		Example: fmt.Sprintf("# Print the client version of Odo\n\n%s",
```

Comment:
would be better to use code similar to https://github.com/redhat-developer/odo/blob/master/pkg/odo/cli/service/create.go#L22-L24 for consistency.

Reply:
Agreed, updated accordingly